Directory.Build.props

```diff
@@ -6,5 +6,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="7.0.0-*" />
     <RdXmlFile Include="rd.xml" />
+    <RdXmlFile Include="Microsoft.EntityFrameworkCore.rd.xml" />
+    <RdXmlFile Include="Microsoft.EntityFrameworkCore.Sqlite.rd.xml" />
```

Comment:
Didn't know this can include multiple `rd.xml` files before 😅

Reply:
I got tired of copy-paste snippets between different projects, so decide to try